website/docs/components/button-set/partials/code/how-to-use.md

```diff
@@ -9,17 +9,57 @@ The basic invocation requires two or more buttons to be provided as children:
 </Hds::ButtonSet>
 ```
 
+Other button-like components can also be used as children such as the `Dropdown`.
+
+```handlebars
+<Hds::ButtonSet>
+  <Hds::Dropdown as |D|>
+    <D.ToggleButton @color="secondary" @text="Select an option" />
+    <D.Title @text="Title Text" />
+    <D.Description @text="Descriptive text goes here." />
+    <D.Interactive @href="#">Add</D.Interactive>
+    <D.Interactive @href="#">Add More</D.Interactive>
+    <D.Interactive @href="#">Add Another Thing Too</D.Interactive>
+    <D.Separator />
+    <D.Interactive @route="components" @icon="trash" @color="critical">Delete</D.Interactive>
+  </Hds::Dropdown>
+
+  <Hds::Button @text="Submit" type="submit" />
+</Hds::ButtonSet>
+```
+
 ### Equal width buttons
 
-If you want to have buttons with equal width, apply a width (via inline style or CSS class) to the `ButtonSet` container and the argument `@isFullWidth={{true}}` to the `Button` components:
+If you want buttons with equal widths, set `@isFullWidth` to `true` on the `Button` components. To constrain the overall max-width, you can also add a width to the `ButtonSet` container (via an inline style or CSS class).
+
+Note: The ButtonSet is full-width (100%) by default.
 
 ```handlebars
-<Hds::ButtonSet {{style width="15rem"}}>
+<Hds::ButtonSet {{style maxWidth="15rem"}}>
   <Hds::Button @text="Save" @isFullWidth={{true}} />
   <Hds::Button @text="Cancel" @color="secondary" @href="https://hashicorp.com" @isFullWidth={{true}} />
 </Hds::ButtonSet>
 ```
 
+Issue: The `Dropdown` does not have an `isFullWidth` option for the outer wrapper. Setting isFullWidth to “true” on the nested ToggleButton does not have the same effect.
```

Comment:
I included this example temporarily to show an issue with the Dropdown:
<img width="859" height="209" alt="image" src="https://github.com/user-attachments/assets/c6e5dce8-f1fd-4eed-a6a1-0c0188761c0f" />

We may want to add an `isFullWidth` option to the Dropdown itself.

Reply:
I'm not seeing examples within consumer code bases but it's still a legitimate use case.

https://github.com/search?q=org%3Ahashicorp+ButtonSet%3E&type=code

Reply:
@heatherlarsen here's what we were seeing: 

![CleanShot 2025-08-01 at 09 19 21@2x](https://github.com/user-attachments/assets/04c4f79a-d2f6-4226-9f55-93c244325932)

![CleanShot 2025-08-01 at 09 22 50@2x](https://github.com/user-attachments/assets/1c8303f8-e85f-4626-b4f3-e3b6395a0034)


What we'd like to be able to ensure: 
![CleanShot 2025-08-01 at 09 24 09@2x](https://github.com/user-attachments/assets/7a25807b-78d4-4efa-aba9-760dcb114c70)

Reply:
Created a follow-on ticket: https://hashicorp.atlassian.net/browse/HDS-5281